src/proximity/index.tsx

```diff
@@ -1,63 +1,21 @@
-import React from 'react'
```

Comment:
You can rename this file `Proximity.unit.tsx`

Reply:
hum, this is the new index file. The unit test file has already been renamed to `Proximity.unit.tsx`

Reply:
@pith is it OK?

Reply:
Yes, sorry stupid comment.